### PR TITLE
Implement tentales-static

### PR DESCRIPTION
**Pull request by dnjstrom**
This is a middleware that simply wraps koa-static and which should be fairly straight forward to use in any project.

This is currently a stand-alone middleware that you have to manually include if needed. This might be fine if we believe most assets should be on a cdn anyway, or we might want to shoot for a more "batteries included" feel where the middleware is included by default.

*Note:*
I noticed that koa-static will automatically serve the gzip or brotli versions if available and supported by the client, so I added compressed versions of hello.html for the lulz 😄 (brotli cuts the size in half, gzip removes about 16%)

screen shot 2018-03-29 at 18 31 55

screen shot 2018-03-29 at 18 32 27

### CODE REVIEW

File: packages/tentales-example-simple/public/hello.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1>Hello!</h1>
+</body>
+
+</html>
```
Comment (emiloberg) [inline packages/tentales-example-simple/public/hello.html]: Nit: Trailing line break

File: packages/tentales-static/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+export type StaticMiddleware = (props: StaticMiddlewareParams) => Middleware
+
+const serveStatic: StaticMiddleware = ({ root, ...otherOptions }) => () =>
+  serve(root, {
+    defer: false,
+    ...otherOptions,
+  })
+
+// serveStatic.displayName = "Static"
```
Comment (emiloberg) [inline packages/tentales-static/src/index.ts:17]: Can we make this return something of type `Middleware` (which is annotated here https://github.com/iZettle/tentales/blob/1c77ba3cad9903b74756dc59aebad8f6651dc8b6/packages/tentales/src/index.d.ts#L80 )

**Review by emiloberg — CHANGES_REQUESTED** [review]
Nice! 

I'm wondering if we should include this in the package, but I can't come to a good conclusion right so lets move it there if needed.

File: packages/tentales-static/src/index.ts (hunk 2 above)
Comment (dnjstrom) [reply packages/tentales-static/src/index.ts:17]: Well, yeah. Isn't it though?

```typescript
export type StaticMiddleware = (props: StaticMiddlewareParams) => Middleware
```

Comment (emiloberg) [reply packages/tentales-static/src/index.ts:17]: Nevermind me

**Review by emiloberg — APPROVED** [review]

**Review by emiloberg — APPROVED** [review]

**Pull request merged**